docs/javascript/v2/advanced-features/peer-metadata.mdx

```diff
@@ -33,6 +36,21 @@ async function toggleRaiseHand() {
 }
 ```
 
+</Tab>
+
+<Tab id='meta-1'>
+
+```jsx
+async function toggleRaiseHand() {
+    const localPeerId = useHMSStore(selectLocalPeerID);
```

Comment:
I don't think we can use it like this. create a component.

Reply:
🤦🏽 Damm silly mistake